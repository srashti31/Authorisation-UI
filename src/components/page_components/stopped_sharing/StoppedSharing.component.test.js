import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
import StoppedSharing from "./StoppedSharing.component";

configure({ adapter: new Adapter() });

describe("<StoppedSharing />", () => {
    it("should match snapshot", () => {
        const snapshot = renderer.create(<StoppedSharing />).toJSON();
        expect(snapshot).toMatchSnapshot();
    });
    it("should render without crashing", () => {
        shallow(<StoppedSharing />);
    });
});
