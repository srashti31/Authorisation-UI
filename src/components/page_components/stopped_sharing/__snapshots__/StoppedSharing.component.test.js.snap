// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StoppedSharing /> should match snapshot 1`] = `
<div
  className="MuiPaper-root stopped-sharing-content MuiPaper-elevation3 MuiPaper-rounded"
>
  <div
    className="stopped-sharing"
  >
    <p
      className="sharing-done-text"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root sharing-done-icon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zM9.29 16.29L5.7 12.7a.9959.9959 0 010-1.41c.39-.39 1.02-.39 1.41 0L10 14.17l6.88-6.88c.39-.39 1.02-.39 1.41 0 .39.39.39 1.02 0 1.41l-7.59 7.59c-.38.39-1.02.39-1.41 0z"
        />
      </svg>
       You have successfully stopped sharing the data below with undefined This information has been archived.
    </p>
    <p
      className="download"
    >
       
      <svg
        className="download-icon"
        fill="none"
        height="1em"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 15v4a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2v-4"
        />
        <polyline
          points="7 10 12 15 17 10"
        />
        <line
          x1="12"
          x2="12"
          y1="15"
          y2="3"
        />
      </svg>
       
      Download conÔ¨Årmation (pdf)
    </p>
  </div>
</div>
`;
